src/details/ArborX_MinimumSpanningTree.hpp

```diff
@@ -391,22 +391,27 @@ void resetSharedRadii(ExecutionSpace const &space, BVH const &bvh,
   // distance between all such pairs. Given that the Morton neighbors are
   // typically close to each other, this should provide a reasonably low bound.
   auto const n = bvh.size();
+  constexpr int num_shifts = 2;
   Kokkos::parallel_for(
       "ArborX::MST::reset_shared_radii",
-      Kokkos::RangePolicy<ExecutionSpace>(space, n - 1, 2 * n - 2),
+      Kokkos::RangePolicy<ExecutionSpace>(space, n - 1, 2 * n - 1 - num_shifts),
       KOKKOS_LAMBDA(int i) {
-        int const j = i + 1;
+        constexpr int shifts[num_shifts] = {1, 2};
```

Comment:
Why did you define it rather than just
```C++
for (int k = 0; k < 2; ++k)
{
  int const j = i + k;

Reply:
Because at some point I tried doing `{1, 2, 3, 5, 10}` or something. Want me to change?

Reply:
Yes.  Also please comment briefly on the performance benefit of doing so.